CHANGELOG.md

```diff
@@ -6,6 +6,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+## 6.7.1
```

Comment:
Maybe 6.8 would be better?
The release started a bug fix but more got added

Reply:
But it's still bug fixes and no new features are being added. So I thought it might be better to keep it 6.7.1?

Reply:
Aside from `opm` upgrade everything is just a bugfix, so `6.7.1` makes sense IMO as we are mostly adding fixes...

I'm not sure if just a new `opm` binary counts as a "feature added" as we just upgraded a dependent component, but I'm fine with 6.8 if you believe this is a "feature" change

Reply:
But honestly... I'm more up to `6.7.1` than `6.8` due to most of it being fixes

Reply:
yeah OPM update was making me inclined towards 6.8. I don't have strong objections to 6.7.1 as well. Happy to go with majority